### ISSUE TITLE
Checkbox on new Canvas file wizard to generate onLoad method

### ISSUE THREAD

**Issue by PhaserEditor2D** [PhaserEditor2D/PhaserEditor#50]
(from user An Tn)

I suggest feature that you can add when create new canvas file is have a check box (called "onLoad") that whether generate onLoad function at the end of create function or not, 
something like this: 

```
function Dice(aGame, aX, aY, aKey, aFrame) {
	....
	Phaser.Sprite.call(this, aGame, aX, aY, pKey, pFrame);
	....
	this.onLoad();
	
}

/* --- end generated code --- */
// -- user code here --
Dice.prototype.onLoad = function(){
	
};
```
This will be useful.

**Comment by PhaserEditor2D** [comment]
Yet I have to think the best way to do this. Maybe a "code template" could be a more general solution.

**Comment by PhaserEditor2D** [comment]
This was solved in the v1.5.1 release:

![image](https://user-images.githubusercontent.com/12187299/44350347-24c3c380-a46d-11e8-9c2b-4f2b43f7fb2f.png)
